lms/static/coffee/src/instructor_dashboard/send_email.coffee

```diff
@@ -15,147 +15,154 @@ create_email_content_table = -> window.InstructorDashboard.util.create_email_con
 create_email_message_views = -> window.InstructorDashboard.util.create_email_message_views.apply this, arguments
```

Comment:
Lots of tab-fixing in this file - to view this file without whitespace changes: https://github.com/edx/edx-platform/pull/12301/files?w=1#diff-84c026f6b037df9822fa7e5caf4ff813